app/javascript/components/BudaForm.vue

```diff
@@ -21,7 +21,7 @@
             <passwordInput fieldId="password" fieldName="password" v-model="password" />
           </div>
           <div>
-            <submitButton :fieldDisabled="false">Sincronizar Buda</submitButton>
+            <submitButton>Sincronizar Buda</submitButton>
```

Comment:
no se si entendí bien, pero según yo la idea ahora es que ahora a través del prop loading se reciba si este botón se debe mostrar o no como cargando, y eso se define a través de vuex con los commits y dispatchs pertinentes o no? En base a eso, en este botón sacaste ese `fieldDisabled` y no lo reemplazaste por `loading` con otra variable sacada del estado porque este botón nunca se muestra cargando?

Reply:
hola! sí, la idea es esa, el loading me permite saber si se está procesando el request o no y de esta manera deshabilitar el botón. En este caso no lo puse porque la carga ocurre de manera instantánea, no en el caso del sign in, signup o payments. Más a futuro habrá que evaluar si es necesario incluirlo.